Library/Homebrew/dependency_collector.rb

```diff
@@ -18,7 +18,7 @@
 class DependencyCollector
   # Define the languages that we can handle as external dependencies.
   LANGUAGE_MODULES = Set[
-    :chicken, :jruby, :lua, :node, :ocaml, :perl, :python, :python3, :rbx, :ruby
+    :lua, :perl, :python, :python3, :ruby
```

Comment:
Looks like it was omitted, yeah. I see no evidence it's been used (though `:lua` is).